sdk/servicebus/Azure.Messaging.ServiceBus/src/Processor/ProcessSessionEventArgs.cs

```diff
@@ -34,6 +34,16 @@ public class ProcessSessionEventArgs : EventArgs
         /// </summary>
         public DateTimeOffset SessionLockedUntil => _sessionReceiver.SessionLockedUntil;
 
+        /// <summary>
+        /// The entity path corresponding to the entity that the session is defined on.
```

Comment:
```suggestion
        /// The path of the Service Bus entity that the message was received from.
```

Based on the [Receiver comment](https://github.com/Azure/azure-sdk-for-net/blob/main/sdk/servicebus/Azure.Messaging.ServiceBus/src/Receiver/ServiceBusReceiver.cs#L34)